src/protocol/webdriver-bidi.ts

```diff
@@ -51,7 +53,9 @@ export type EventData =
   | LogEvent
   | NetworkEvent
   | ScriptEvent;
-export type Extensible = Record<string, unknown>;
+export type Extensible = {
+  [key: string]: any;
```

Comment:
If possible (if compilation doesn't fail), `unknown` would be preferred.

https://google.github.io/styleguide/tsguide.html#any

Reply:
Yea, I'll update the parser to use unknown.